src/mint_json.cr

```diff
@@ -40,7 +41,13 @@ module Mint
           "node" => node(exception),
         }
       end
+
       parse_root
+
+      # TODO: Enable when releasing 0.13.0!
```

Comment:
Only enable this until `0.13.0` so it would not cause issues with testing things until then.